src/index.html

```diff
@@ -7,8 +7,64 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="#home" class="logo"><img src="./images/logo.png" alt="logo" ></a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#Apple"
+              class="nav_link is-active">
+            Apple
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#Samsung" class="nav_link">Samsung</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#Smartphones" class="nav_link">Smartphones</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#Laptops & Computers" class="nav_link"
+               data-qa="hover">Laptops & Computers</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#Gadgets" class="nav_link">Gadgets</a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#Tablets"
+              class="nav_link"
+             >
+              Tablets
+            </a>
```

Comment:
```suggestion
            <a href="#Tablets" class="nav_link">
              Tablets
            </a>
```
elements with two attributes can be in one line